src/Snap.h

```diff
@@ -109,10 +107,9 @@ class AUDACITY_DLL_API SnapManager
    SnapPointArray mSnapPoints;
 
    // Info for snap-to-time
-   NumericConverter mConverter;
```

Comment:
The commit that removes this can replace `#include "NumericTextCtrl.h"` with "ComponentInterfaceSymbol.h", lowering this guy in the dependency graph.

That will require other `#include`s added in other places that depended on transitive inclusions.